### ISSUE TITLE
__mocks__ style mocking of organization modules

### ISSUE THREAD

**Issue by KristjanTammekivi** [theKashey/rewiremock#121]

**Comment by theKashey** [comment]
https://github.com/theKashey/rewiremock/blob/b350aff1c4fe05d4329ca6f4e4033d29d240d794/src/plugins/__mock__.js#L11-L22
This moment should check more than one directory.

**Comment by KristjanTammekivi** [comment]
I added logging to autoMock to log out mockName if mockName includes our org name/package name and it only logged out
/...project root.../node_modules/@ourorg/package/dist/__mocks__/index.js

**Comment by theKashey** [comment]
Yes. It is trying to resolve `__mock__` only in one location, that's why you've created this issue.
It can be fixed, but it is yet to fix.

Actually you can just reimplement this plugin in user space in a way it will work best for you, or wait a few days untill I'll get time to fix it.
